src/baseline_automatic_pipeline_system/eval_auto_pipeline.py

```diff
@@ -81,7 +91,7 @@ def eval_auto_pipeline(input_folder,gold_folder):
 
     print 'Predicate mentions(F1) = %.3f Recall = %.3f Precision = %.3f  ' % (predicate_mention_score[0],predicate_mention_score[1],predicate_mention_score[2])
 
-    print 'Entity mentions(F1): %.3f' % entity_mention_score[0]
+    print 'Entity mentions(F1): %.3f Recall = %.3f Precision = %.3f' % tuple(entity_mention_score[:3])
```

Comment:
What numbers are you getting here?
Maybe you can post that to the PR description?
I think we also had a file logging our performance with each update?

Reply:
you're right. updated numbers in the file.